### DISCUSSION TITLE
Mobile app cloud storage

### DISCUSSION THREAD

**Discussion by guydunigo** [BoostIO/BoostNote.next-local#54] [Ideas]
Hello,
I don't know if you planned to work on a mobile app of this version of BoostNote as well. 
If you have, it could be nice to have a feature similar to the old legacy boostnote mobile app which could connect to DropBox to access boostnote-local notes.

**Comment by Komediruzecki** [comment]
Hi,

sorry for the delay in response, somehow missed the question.

I think it would be a good feature, we can leave this discussion as interest gathering, but for advanced syncing capabilities, cloud spaces should be used.
